### PR TITLE
pug outfit sprites

### PR DESCRIPTION
**Pull request by comnom**
Edit 12/1:
All T1 outfits complete.

### CODE REVIEW

**Comment by comnom** [comment]
.blend and .xcf here:
[pugzapperassets.zip](https://github.com/endless-sky/endless-sky/files/609476/pugzapperassets.zip)

**Comment by jafdy** [comment]
The @2x files go with the high dpi plugin. [here](https://github.com/endless-sky/endless-sky-high-dpi)

**Comment by Amazinite** [comment]
The image names should have spaces in them.

**Comment by jafdy** [comment]
Just to link them: #1834

**Comment by Bladewood** [comment]
Hmm... not enough red.
Looks to be made of gold rather than Pug ship materials.

Oh, wait, this is for tier 1, not tier 3.
In that case, I think...

1. There should be a hexagonal [grid, pattern], just like on the ships.
2. The red should be a little more spread out.
3. The prongs should be grey, like the spikes on the ships.

Because consistency is nice.

**Comment by Pointedstick** [comment]
I like 'em! Nice work.

**Comment by endless-sky** [comment]
I really like how these turned out!

There are 11 different "Tier 1" Pug outfits - 4 weapons, 6 engines, and the biodefenses. Ideally, I'd like the weapons to share a common design aesthetic, so I'd like to hold off on merging this in until you've settled on a final design and have sprites for the other two weapons, too, if you have time to do that.

(Please don't take that as me saying I don't like your work - these look really cool and Pug-like, I'm just worried that if you end up not having time to make more, it'd be hard for someone else to mimic your style in order to make other sprites that match it.)

**Comment by comnom** [comment]
@Amazinite fixed in https://github.com/endless-sky/endless-sky/pull/1837/commits/6b87936fa57f55c2153121a73fcc7c54a248c89d

@endless-sky I gotcha. I like to jump around so all the weapons don't end up the same from doing one right after the other, so I may as well just do it all in here. I intend on doing the whole set.

**Comment by Bladewood** [comment]
The Akfar engines are... scary...

**Comment by comnom** [comment]
>The Akfar engines are... scary...

The more I look at the ship art, the more it reminds me of a Malboro from FF ('cept yellow). So that may be influencing it a bit. ;)

**Comment by Amazinite** [comment]
Those engines
are
amazing.

**Comment by Lineth** [comment]
Indeed.

Only if Pug engine stats weren't pants. :(

**Comment by Bladewood** [comment]
interesting.

so do you intend to add "tiny" Pug engines?

**Comment by comnom** [comment]
@Bladewood No, I won't be adding anything besides the artwork for the existing T1 outfits. I'm sure before all is said and done the Pug will have one of the largest sets of outfits, but that's all for the dev to decide. Although there is likely little need for more T1 outfits.

**Comment by Bladewood** [comment]
Pug engines seem to be [universal] among the Pug tiers, given that even Arfectas use them, so there's no harm in adding moar engines~

**Comment by Lineth** [comment]
Cormet seem a little too similar to the Akfars imo.

**Comment by comnom** [comment]
@Lineth Yea that's pretty much the standard with all the engines. I even follow it in my mod. Small changes, or adding a second or third pair. I was going to put some spikey bits on the little guys but it threw off the symmetry, so I'm saving it for the large set.

I suppose it's not a rule, but it's what people expect to see I think, alongside all the others.

**Comment by Bladewood** [comment]
Hm... I was envisioning the biodefenses as [pouches, sacs], like the ones [ravagers](http://masseffect.wikia.com/wiki/Ravager) have.

**Comment by Lineth** [comment]
Will you be making images for the Gridfire and Tier 3 AM turrets?

**Comment by comnom** [comment]
@Bladewood That's a pretty cool idea, however I'm not nearly good enough to pull off something that organic. Organic modeling is an order of magnitude more difficult than hard surface modeling. I'm may make an attempt, but I'm pretty sure nothing good will come from it.

@Lineth I'll be doing the gridfire, seeker, lohmar, and the T1 anti-missile. I'm not sure but I think the T3 is more of a placeholder right now, and is probably awaiting other outfits of that class.

**Comment by Bladewood** [comment]
Idea:
Gridfire turrets and/or tier 3 anti-missile turrets should resemble jump drives, but with red lights instead of magenta, not necessarily a hexagonal base, and a bit of a wackier design on top.

File: data/pug.txt [hunk 1]
```diff
@@ -103,7 +103,7 @@ outfit "Pug Seeker"
 outfit "Pug Anti-Missile"
 	category "Turrets"
 	cost 350000
-	thumbnail "outfit/unknown"
+	thumbnail "outfit/pug antimissile"
```
Comment (Bladewood) [inline data/pug.txt]: should be `thumbnail "outfit/pug anti-missile"`, for consistency.

**Comment by Bladewood** [comment]
Was not what I was anticipating, but that looks pretty cool and scary!

**Comment by comnom** [comment]
@Bladewood fixed in https://github.com/endless-sky/endless-sky/pull/1837/commits/9125f5eafc2c91fe1ff99d86ab146ca993c33412

**Comment by Admiral-Danforth** [comment]
Just an idea, but they should change color depending on which stage you are at:
Merchant: Stock/Blue
Free World: Greenish 
Almost like they were spray painted to match you ship.
(Just weapons)

**Comment by Bladewood** [comment]
Outfits cannot be swizzled.
It's better for the Pug weapons to be Pug-colored.

**Comment by Amazinite** [comment]
The engine names should probably follow the same naming scheme as everything else. 
Akfar -> "small pug (thruster or steering)
Cormet -> "medium pug..."
Lohmar -> "large pug..."

**Comment by comnom** [comment]
Wait, the Gridfire is T3. With the addition of the Seeker, I'm finished.

@Amazinite Maybe but that's outside the scope of this PR.

**Comment by Amazinite** [comment]
Not really.

**Comment by Wrzlprnft** [comment]
I like what you did with that. Ill check how the outfitter page looks with this later. Maybe it needs a little color variation,but i haven't seen the images next to esch other in the outfitter yet.

**Comment by Amazinite** [comment]
I'm talking about the engine image names, btw.

**Comment by comnom** [comment]
> I'm talking about the engine image names, btw.

Ah, I thought you meant an outfit name change. That aside, I'd still like to stick to just adding the requested artwork in this PR.

**Comment by Lineth** [comment]
Do you plan on doing some outfit sprites for the upcoming coalition? Or wait until they're implemented?

**Comment by Amazinite** [comment]
Well the Coalition aren't even fully developed yet, let alone in a release.

**Comment by endless-sky** [comment]
These are awesome. Sorry for the delay in merging them in - I was too busy with some of my 21st-century commitments and got behind on my 31st-century ones. :)

**Comment by Amazinite** [comment]
🎆 🎆 🎆 🎆 🎆

**Comment by endless-sky** [comment]
Also, @comnom, what's your real name (or whatever name you want to go by) to put for those sprites in the copyright file?

**Comment by comnom** [comment]
@endless-sky Frederick Goy IV for the copyright.
> Sorry for the delay in merging them in

No sweat man, I'm just happy to have contributed something.

**Comment by endless-sky** [comment]
I've listed them as using the same license as the rest of the game assets, CC-BY-SA 4.0. (Meaning, anyone can use or modify those images as long as they credit you and allow others to use their copies of the images under the same terms.) Let me know if you'd rather use a different license.

**Pull request merged**